### PR TITLE
Navigation JobListView to JobDetailView

### PR DESCRIPTION
**Pull request by sact1909**
## Navigation to from JobListView to JobDetailWiew - Ref #27 ##

• Remove BindingContext from XAML file, it was assigned in the back end in the cs files from JobsListView and JobDetailView 
• Send Job Link Parameter from JobsListView to JobDetailView

### Ways to assign ViewModel to BindingContext in the Back End File ###
```md
BindingContext = new MyViewModel(<ParamToSend>);
```

### Visual aids ###

| Preview| 
|---|
|navigationlistview|

### CODE REVIEW

File: src/App/ViewModels/JobDetailViewModel.cs [hunk 1]
```diff
@@ -15,18 +15,18 @@ public class JobDetailViewModel : INotifyPropertyChanged
...
+        public JobDetailViewModel(string Link)
         {
 
-            new Action(async () => await LoadData())();
+            new Action(async () => await LoadData(Link))();
 
             OpenJobWebURLCommand = new Command<string>(OpenJobWebURL);
         }
-        public async Task LoadData()
+        public async Task LoadData(string link)
```
Comment (claudiosanchez) [inline src/App/ViewModels/JobDetailViewModel.cs]: The name of the method should end in Async -> LoadDataAsync

File: src/App/ViewModels/JobsListViewModel.cs [hunk 2]
```diff
@@ -16,10 +17,15 @@ class JobsListViewModel : INotifyPropertyChanged
     {
         //public AppConstant ServiceConstant = new AppConstant();
         public JobListModel JobsList { get; set; }
-        public JobsListViewModel()
+        public Command CallDetailView { get; set; }
```
Comment (claudiosanchez) [inline src/App/ViewModels/JobsListViewModel.cs]: The name of the property should end in "Command" ->i.e.:  CallDetailScreenCommand

**Review by claudiosanchez — CHANGES_REQUESTED** [review]

File: src/App/ViewModels/JobsListViewModel.cs (hunk 2 above)
Comment (sact1909) [reply src/App/ViewModels/JobsListViewModel.cs]: corregido

File: src/App/ViewModels/JobDetailViewModel.cs (hunk 1 above)
Comment (claudiosanchez) [reply src/App/ViewModels/JobDetailViewModel.cs]: @sact1909 ^

Comment (sact1909) [reply src/App/ViewModels/JobDetailViewModel.cs]: Listo, tambien le agregue lo mismo a otros metodos que encontre que tenian el mismo caso...

**Review by claudiosanchez — APPROVED** [review]

**Pull request merged**